sdk/go-sdk/demo/file/client.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 Layotto Authors
```

Comment:
There are some existing golang demos under `demo` packages:
<img width="249" alt="image" src="https://user-images.githubusercontent.com/26001097/227721463-8adfee2c-bcf2-4867-8c56-528069ade46c.png">
Maintaining two demo packages is hard. So I recommend that we keep only one demo package. How about refactoring the existing `demo` package instead of adding a new one?